### DISCUSSION TITLE
Unit tests

### DISCUSSION THREAD

**Discussion by pepijndik** [lucasgiovanny/laravel-prestashop#10] [Ideas]
I have created a basic for the test on the v2-dev branch But it gets a lot of errors which i dint get rid of so maybe @lucasgiovanny  can you work on this further?

**Comment by lucasgiovanny** [comment]
Hi @pepijndik! Yesterday was a very busy day at work, I wasn't able to do much with the project. I just started working on organizing the structure of the tests on the branch `v2-dev` so I got this. :)

**Reply by pepijndik** [reply]
No problem, i probally wont have time today to futher work on it.

**Reply by pepijndik** [reply]
@lucasgiovanny  are you going to cover some test or should I made some tests and which testing framework do you prefer.
Pest v2 or Phpunit testing :-D

**Reply by lucasgiovanny** [reply]
> @lucasgiovanny are you going to cover some test or should I made some tests and which testing framework do you prefer.
> Pest v2 or Phpunit testing :-D

I'm working on the documentation and tests this week. I'm going to use Pest PHP v2.

Maybe we can have something stable next week to release.
